cw_wownero/lib/api/transaction_history.dart

```diff
@@ -252,15 +253,13 @@ Future<PendingTransactionDescription> createTransactionMultDest(
 
 class Transaction {
   final String displayLabel;
-  String get subaddressLabel => wownero.Wallet_getSubaddressLabel(wptr!, accountIndex: 0, addressIndex: 0);
-  String get address => wownero.Wallet_address(
-    wptr!,
+  late String subaddressLabel = wownero.Wallet_getSubaddressLabel(wptr!, accountIndex: 0, addressIndex: 0);
+  late String address = getAddress(
```

Comment:
shouldn't this take the accountIndex and addressIndex instead of 0s?

also why did you change the getters to late in some files?
late will throw an exception if it tried to initialize the variable again, and it's not even final, so it won't give any error if someone tries to set its value again, why not leave them as getters?

Reply:
fixed the 0 and added proper accountIndex/addressIndex variable there. I've made them final as well - the idea is for the code to be future proof - with `late` it won't getn initialized until it is being used - and next uses are from memory instead of constantly calling a function - which should be more performant.